### ISSUE TITLE
Stop turbogears from printing Objects

### ISSUE THREAD

**Issue by kentbower** [kbower/tickettest1#121]
_Imported from trac ticket 121.
Created by **kbower**
Opened in trac: Oct 31, 2009, 1:58:29 AM
Last modified in trac: Oct 31, 2009, 2:12:55 AM_
***
Apache's error_log or nosetests output shows this:

```
[Sat Oct 31 01:17:34 2009] [error] <pylotengine.controllers.error.ErrorController object at 0x2aaab64e14d0>
[Sat Oct 31 01:17:34 2009] [error] <pylotengine.controllers.root.RootController object at 0x2aaab666bd90>
[Sat Oct 31 01:17:34 2009] [error] <pylotengine.controllers.error.ErrorController object at 0x2aaab64e14d0>
[Sat Oct 31 01:17:34 2009] [error] <pylotengine.controllers.root.RootController object at 0x2aaab666bd90>
.
.
.
```

**Comment by kentbower** [comment]
_**kbower** trac comment on Oct 31, 2009, 1:59:28 AM:_

see [http://trac.turbogears.org/changeset/6592]

**Comment by kentbower** [comment]
_**kbower** trac comment on Oct 31, 2009, 2:12:55 AM:_

(In [[134]](../commit/d38e189fa6d6c6d59298cee6aa0faa17ac26b4b5)) I removed the print command from the tg source (see [http://trac.turbogears.org/changeset/6592])

changed version in release.py and did 'python setup.py build' and 'python setup.py install'

Then removed the build and dist directories and re-zipped up the source with a new directory to match the patched version number.  

Last, update src/appserver/setup.py to force a re-install on next upgrade.

This fixes #121

**Comment by kentbower** [comment]
_**kbower** trac status change on Oct 31, 2009, 2:12:55 AM:_ **closed**
